tasks/task400_paws_paraphrase_classification.json

```diff
@@ -6,7 +6,10 @@
         "PAWS (\"https://arxiv.org/abs/1904.01130\")"
     ],
     "Categories": [
-        "Classification"
+        "Classification -> Pairwise Classification"
```

Comment:
Paraphrasing
Text comparison
Maybe argument reasoning
Do we need Pairwise Classification or it's the combination of classifications and text comparison?

Reply:
Good point! I hadn't noticed "Text Comparison" in the taxonomy. I agree that this task could be assigned to ["Classification", "Text Comparison"]. However, "Classification -> Pairwise classification" could capture more tasks. For example, check [task198_mnli_domain_classification.json](https://github.com/allenai/natural-instructions-expansion/blob/fd29662779eb64522dc1c5e0acbcb085ecc7c86b/tasks/task198_mnli_domain_classification.json). It is considered as a pair-wise classification task as the input consists of a pair of sentences but it should not be classified under "Text Comparison" as there's no notion of comparison in this task. So, we could possibly keep pairwise classification as another category in the taxonomy. What do you think?

Reply:
Yeah, pair-wise classification makes sense here.
Also, I'm in favor of dropping task198_mnli_domain_classification.json from the tasks. The genres in this task refer to the source of collecting data, and it's hard to distinguish them from the sentences.